connectors/AnthropicConnector/src/main/java/de/l3s/interweb/connector/anthropic/entity/CompletionBody.java

```diff
@@ -0,0 +1,78 @@
+package de.l3s.interweb.connector.anthropic.entity;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+import de.l3s.interweb.core.completion.CompletionQuery;
+import de.l3s.interweb.core.completion.Message;
+import de.l3s.interweb.core.completion.Message.Role;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@RegisterForReflection
+public final class CompletionBody {
+
+    private List<CompletionMessage> messages;
+
+    private String model;
+    
+    private String system;
+
+    private Double temperature;
+
+    @JsonProperty("top_p")
+    private Double topP;
+
+    @JsonProperty("max_tokens")
+    private Integer maxTokens;
+
+    public CompletionBody(String model, CompletionQuery query) {
+        this.model = model;
+        System.out.println("model: " + model);
+        if (this.model == null) {
+            throw new IllegalArgumentException("model must be set");
+        }
+
+        this.messages = query.getMessages().stream().filter(
+            m -> m.getRole() != Role.system
```

Comment:
it should be formatted a little differently, e.g.
```
this.messages = query.getMessages().stream()
         .filter(m -> m.getRole() != Role.system).map(CompletionMessage::new).toList();
this.system = query.getMessages().stream()
        .filter(m -> m.getRole() == Role.system)
        .findFirst()
        .map(Message::getContent)
        .orElse(null);
        ```
  Something like this, you can find some examples somewhere in server module.